ruby/ql/test/library-tests/controlflow/graph/Cfg.expected

```diff
@@ -811,7 +811,7 @@ case.rb:
 #-----|  -> [ ..., * ]
 
 #   35| [ ..., * ]
-#-----| match -> x
+#-----| false, match, true -> x
```

Comment:
Is this actually right? I wonder where the `true` and `false` edges come from.

Reply:
No, this is a bug. It may be addressed already by @hmac's [CFG changes](https://github.com/github/codeql/pull/11114), but in all cases, I think we should handle it follow-up.

Reply:
Perfect!